src/tests/ftest/util/soak_utils.py

```diff
@@ -1005,6 +1005,7 @@ def create_ior_cmdline(self, job_spec, pool, ppn, nodesperjob, oclass_list=None,
             if api == "POSIX-LIBPIL4DFS":
                 env["LD_PRELOAD"] = os.path.join(self.prefix, 'lib64', 'libpil4dfs.so')
                 env["D_IL_REPORT"] = "1"
+                env["D_IL_INTERCEPTION_ON"] = "1"
```

Comment:
I'm not sure if we want to do this in soak testing. It might be better for soak to run similar to production/users

Reply:
What applications does soak run? fio, ior, mdtest, and lammps etc? This env is supposed to make no difference. I will remove it as you suggested. Thank you!

Reply:
The env is removed in soak test. Thank you!